third_party/blink/renderer/modules/cobalt/h5vcc_metrics/h_5_vcc_metrics.idl

```diff
@@ -49,12 +49,13 @@ interface H5vccMetrics {
   // behavior."
   // This API is now Promise-based.
   [CallWith=ScriptState, RaisesException] Promise<boolean> isEnabled();
```

Comment:
Let's change this to a synchronous function instead to match the c25 API

Reply:
Gotcha, next commit has only one method here, `bool isEnabled();`, essentially looking like Cobalt25.